api-server/rhelai-install.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# -*- indent-tabs-mode: nil; tab-width: 2; sh-indentation: 2; -*-
+
+# Install script for the API server
+## dependencies: go, git
+## required ENV vars: TAXONOMY_PATH, BASE_DIR
+
+set -x
+set -e
+set -o pipefail
+
+### installations
+
+rpm-ostree install gcc glibc-devel
+systemctl reboot
+export CGO_ENABLED=1
```

Comment:
Console a message that the system needs to reboot and to re-run the script after it boots.